cli_tools/gce_image_publish/publish/types.go

```diff
@@ -0,0 +1,82 @@
+//  Copyright 2017 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// publish holds the publish structs
+package publish
+
+import (
+	"context"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+	"google.golang.org/api/compute/v1"
+)
+
+//TODO(krzyzacy): consider move all publish related functions into the publish package
+
+// Publish is a publish workflow for gce_image_publish
```

Comment:
The description is a bit misleading: Publish is not actually a Daisy workflow but holds info that is used when creating a workflow.

Reply:
done